### PR TITLE
wishlist-page

### PR DESCRIPTION
**Pull request by Shailseen**
image

### CODE REVIEW

File: pages/homepage/wishlist-page.css [hunk 1]
```diff
@@ -0,0 +1,40 @@
+/* @import url(/pages/homepage/cart-page.css); */
```
Comment (Aditi0505) [inline pages/homepage/wishlist-page.css]: Please remove this if not needed.

**Review by Aditi0505 — APPROVED** [review]
LGTM

File: pages/homepage/wishlist-page.css (hunk 1 above)
Comment (Shailseen) [reply pages/homepage/wishlist-page.css]: oh yeah i forget about that.

**Review by Creator-singh — APPROVED** [review]
looks good to me

**Pull request merged**